src/ParkChecklist/ParkChecklist.js

```diff
@@ -123,161 +93,119 @@ const ParkChecklist = ({ apiLink }) => {
-                {"<"}
-              </button>
-            )}
-            <span>{`${currentPage}/${Math.ceil(
-              parks.length / parksPerPage
-            )}`}</span>
-            {currentPage < Math.ceil(parks.length / parksPerPage) && (
-              <button onClick={() => handlePageChange(currentPage + 1)}>
-                {">"}
-              </button>
-            )}
-            {currentPage !== Math.ceil(parks.length / parksPerPage) && (
-              <button
-                onClick={() =>
-                  handlePageChange(Math.ceil(parks.length / parksPerPage))
-                }
-              >
-                {">>"}
-              </button>
-            )}
-          </div>
-        </>
-      ),
-    },
-    {
-      path: `/login`,
-      element: (
-        <div>
-          <Login loggedIn={loggedIn} setLoggedIn={setLoggedIn} />
+  return (
+    <div className="park-checklist-container">
+      {loading ?? <p className="loading-message">Loading Parks...</p>}
+      <div className="below-header-box">
+        <div className="state-selector">
+          <select onChange={handleStateChange} value={selectedState}>
+            <option value="">Select a State</option>
+            {states.map((stateCode) => (
+              <option key={stateCode} value={stateCode}>
+                {stateCode}
+              </option>
+            ))}
+          </select>
+          <button onClick={handleGoClick}>Go!</button>
+        </div>
+      </div>
+      <div className="sorting-options">
+        <div className="sort-by-state">
+          <StateSort
+            states={states}
+            selectedStateParks={selectedStateParks}
+            handleStateCheckboxChange={handleStateCheckboxChange}
+          />
         </div>
-      )
-    },
-    {
-      path: `/Parks/${selectedState}`,
-      element: showResults ? (
-        <SelectedStateParks
-          parks={parks}
-          selectedState={selectedState}
-          fetchParkDetails={fetchParkDetails}
+        <DesignationSort
+          getUniqueDesignations={getUniqueDesignations}
+          selectedDesignations={selectedDesignations}
+          handleDesignationCheckboxChange={handleDesignationCheckboxChange}
         />
-      ) : null,
-    },
-    {
-      path: `/Parks/${selectedState}/${park_code}`,
-      element: selectedState ? <ParkDetails foundPark={foundPark} /> : null,
-    },
-    {
-      path: "/*",
-      element: <Navigate to="/" />,
-    },
-  ]);
-
-  return (
-    <div
-      className="park-checklist-container"
-      // style={{ backgroundImage: `url(${page})` }}
-    >
-      <Link to="/" className="header-link">
-        <h1 className="page-title">National Park Service Service</h1>
-      </Link>
-      {loading ? <p className="loading-message">Loading Parks...</p> : routes}
+      </div>
+      <h2>{`${totalChecked}/${parks.length} parks visited`}</h2>
+      <h2>Parks Checklist</h2>
+      <ul className="checkbox-list">
```

Comment:
I see what you mean here on this code. Lex may have a better understanding of it since it is his original code. I haven't had any luck simplifying it, but with your refactor of everything else, it is much more readable on this file.